### ISSUE TITLE
relocate spacy data

### ISSUE THREAD

**Issue by hijoe320** [explosion/spaCy#446]
Hi,

The size of data for English is around 500MB, which almost exceeds my personal quota in a non-privately managed system.

NLTK defines an ENV variable like NLTK_DATA, I notice you also have SPACY_DATA in test scripts, but no longer works in new version.

Any way to relocate the data folder to other place? 

Many thanks!

**Comment by bittlingmayer** [comment]
You can install to a different path on the system with `data_path`.

Instead of invoking download.py with `python -m spacy.en.download`, you should invoke python code that does a slightly modified version of what [download.py](https://github.com/spacy-io/spaCy/blob/master/spacy/download.py) does:

> import sputnik
> import spacy
> sputnik.install('spacy', spacy.about.__version__, 'en_default', data_path=data_path)
> nlp = spacy.load('en_default', via=data_path)

(Copied from the resolution to https://github.com/spacy-io/spaCy/issues/319)

If it is needed by enough clients, perhaps it should be an option to download.py, like `--force` is?

**Comment by renuk16** [comment]
Is there a way to install a language from a tarball instead of pointing to a URL? I have installed spacy on a cluster that is behind firewall and cannot connect to the internet. I saw on other issue comments that I cannot copy the language model from my current installation. The above sputnik suggestion automatically tries to download from the internet and fails. Is there a resolution for this?

Thanks!
